### DISCUSSION TITLE
Scanning port 6379 stuck using nuclei, CVE-2020-11981

### DISCUSSION THREAD

**Discussion by xiaotianstudy** [projectdiscovery/nuclei#4224] [Q&A]
![944af131d55a2feb7a0b76453d4450a](https://github.com/projectdiscovery/nuclei/assets/125735128/1d22e5b3-ff34-412e-b9f5-6a544919dbff)
Scanning port 6379 stuck using nuclei.The template used is CVE-2020-11981.

**Comment by xiaotianstudy** [comment]
I need help. Please!

**Comment by ehsandeep** [comment]
@xiaotianstudy can you share more info to reproduce this? target / cmd

**Comment by xiaotianstudy** [comment]
@ehsandeep The program should be stuck at one of the templates, is there a timeout time parameter for the template?

**Comment by ehsandeep** [comment]
@xiaotianstudy you can exclude `-eid CVE-2020-11981` to exclude this template, and nuclei should timeout as default, for more information, we need to reproduce this issue to confirm/validate this bug.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
